enclave.dev.toml

```diff
@@ -12,7 +12,8 @@ forward_proxy_protocol = false
 trusted_headers = []
 
 [egress]
-enabled = false
+enabled = true
+destinations = ["*.auth.zelus.io","*.amazonaws.com"]
```

Comment:
is it possible to narrow this down to point at the specific amazon APIs that we need? Otherwise it's conceivably possible to do egress to arbitrary EC2 instances, which should be prevented.